#!/usr/bin/env python
PACKAGE = "jsk_uav_forest_perception"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("enable",    bool_t,    0, "Enable", False)
gen.add("color_hsv_h_min",    int_t,    0, "H lower bound", 0,  0, 255)
gen.add("color_hsv_h_max",    int_t,    0, "H upper bound", 113,  0, 255)
gen.add("color_hsv_s_min",    int_t,    0, "S lower bound", 202,  0, 255)
gen.add("color_hsv_s_max",    int_t,    0, "S upper bound", 255,  0, 255)
gen.add("color_hsv_v_min",    int_t,    0, "V lower bound", 2,  0, 255)
gen.add("color_hsv_v_max",    int_t,    0, "V upper bound", 216,  0, 255)

exit(gen.generate(PACKAGE, "jsk_uav_forest_perception", "ColorRegionDetector"))
